packages/extension-chakra-storefront/src/pages/index.tsx

```diff
@@ -35,9 +35,6 @@ const ProductDetail = loadable(() => import('overridable!./product-detail'), {fa
 const ProductList = loadable(() => import('overridable!./product-list'), {
     fallback
 })
-const Wishlist = loadable(() => import('overridable!./account/wishlist'), {
-    fallback
-})
```

Comment:
Why are we removing the wishlist?

Reply:
It turns out that the Wishlist route has not been matched/used. It's been taken care of by the Account page: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/92fb26a02bd13789eaf9a7d3855edb5c894ebdfa/packages/extension-chakra-storefront/src/pages/account/index.jsx#L211-L213